### DISCUSSION TITLE
Error with Network Shares

### DISCUSSION THREAD

**Discussion by misaelr78** [slskd/slskd#944] [Q&A]
Hi, 

Im having issues trying to access CIFS and NFS Shares from SLSKD. The shares are good. I can access from other devices but no matter what setting I set in the config page, never work. Any susgestion?

![image](https://github.com/slskd/slskd/assets/92052900/d4248005-bda7-455e-80d5-267c5936378d)

**Comment by jpdillingham** [comment]
Hi @misaelr78, apologies for not following up (doing a year-end review of things!); is this still an issue?

One thing that jumps out is that the error message appears to have omitted one of the leading forward slashes, which might mean the path is getting mangled somewhere and that the application isn't able to identify this as a network path.  I'll see if I can set up a reproduction.
